collections/includes_value.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Returns whether the given value is part of the collection.
+ *
+ * Example:
+ * ```ts
+ * import { includesValue } from "./includes_value.ts";
+ * import { assertEquals } from "../testing/asserts.ts";
+ *
+ * const input = {
+ *   first: 33,
+ *   second: 34,
+ * };
+ *
+ * assertEquals(includesValue(input, 34), true);
+ */
+
+export function includesValue<T>(
```

Comment:
```typescript
includesValue({ k: NaN }, NaN); // gives false
```
Consider using `Object.is` instead? Or document this behavior :-)

Reply:
I've added a check for NaN values using `Number.isNan()` and a test for it.